modules/util/localizer.ts

```diff
@@ -0,0 +1,45 @@
+import { utilArrayUniq } from './array';
+
+function collectLocale({ script, region, language }: Intl.Locale) {
```

Comment:
Some common tags require a variant code, such as the `tarask` in [`name:be-tarask=*`](https://taginfo.openstreetmap.org/keys/name:be-tarask). `maximize()` returns `be-Cyrl-BY-tarask`. There’s a `variant` property but only Firefox supports it.

This is an edge case that we can ticket out as tail work. iD doesn’t currently have any localizations that require a variant. The closest is our “ar-AA” locale, which should probably be just “ar”. (I think Transifex made up a country code for the Unitag variant.)

Reply:
I've opened #11731 to track this